packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -56,6 +56,7 @@ const applyTransparentBackplateStyles: CSSRules<DesignSystem> = {
         boxShadow: "none",
         "& $button_contentRegion::before": {
             background: neutralForegroundRest,
+            height: toPx<DesignSystem>(focusOutlineWidth),

```

Comment:
With this change, what is the difference between a hyperlink and our button component? Doesn't button just address hyperlink scenarios now?

Reply:
They were always visually consistent. As I understood, Hyperlink we more for use inline as an actual href and would take font styling from its parent, and Button is more for an actual semantic button that looks like a link. That is, not inline with other content.

Reply:
Button supports either an anchor or semantic button - these can look the same and function differently. The delta was that there was no underline on the button by default as there is no accessibility requirement for it as it is not inline with other text (as hypertext is). So, there was a visual delta between the two intentionally; the underline by default was on hypertext only as it is intended to be used inline with text.

Reply:
**Button:**
<img width="1591" alt="Screen Shot 2019-06-21 at 10 43 07 AM" src="https://user-images.githubusercontent.com/13071055/59941175-6b79a580-9411-11e9-8a5b-a3e0da7594bf.png">

**Hypertext**
<img width="1591" alt="Screen Shot 2019-06-21 at 10 44 50 AM" src="https://user-images.githubusercontent.com/13071055/59941269-a2e85200-9411-11e9-868d-d58334aea9ac.png">

Reply:
Got it. It looks like my comment is misleading. There is still the same underline setup for the two components in default state. The material change is that _when_ there is an underline it is 1px _unless_ it's focus state, at which it's 2px.

Reply:
Gotcha - I'll pull down just to make sure I'm seeing things correctly and sign off :thumbsup:

Reply:
Title and comments updated.